src/platform/windows/display_base.cpp

```diff
@@ -1037,15 +1042,32 @@ namespace platf::dxgi {
 namespace platf {
   std::shared_ptr<display_t>
   display(mem_type_e hwdevice_type, const std::string &display_name, const video::config_t &config) {
+    if (config::video.capture == "wgc") {
+      if (hwdevice_type == mem_type_e::dxgi) {
+        auto disp = std::make_shared<dxgi::display_wgc_vram_t>();
+
+        if (!disp->init(config, display_name)) {
+          return disp;
+        }
+      }
+      else if (hwdevice_type == mem_type_e::system) {
+        auto disp = std::make_shared<dxgi::display_wgc_ram_t>();
+
+        if (!disp->init(config, display_name)) {
+          return disp;
+        }
+      }
+      // if not supported, fall back to Desktop Duplication API
```

Comment:
What's the minimum build version of Windows 10 that supports this method?

Reply:
It's supported to some degree since Win10 1803, but it isn't full featured enough for our usage until Win11 and Server 2022.

Reply:
Do you think this will work without a DirectX11 gpu, and only a virtual display?

If possible I'd like to try removing these lines to see if the encoder will work in CI.

- https://github.com/LizardByte/Sunshine/blob/89e8b9628c9e4d763443fe871c45afe1fcf39396/.github/workflows/CI.yml#L864
- https://github.com/LizardByte/Sunshine/blob/89e8b9628c9e4d763443fe871c45afe1fcf39396/.github/workflows/CI.yml#L1012

Reply:
Same issue :/

https://github.com/LizardByte/Sunshine/actions/runs/8459022198/job/23174508060?pr=2306#step:9:105